spec/game_spec.rb

```diff
@@ -0,0 +1,103 @@
+require './lib/classes.rb'
+  	
+RSpec.describe Check_win do
+	describe "#check win" do
+		win= Check_win.new
+		it "it returns true if there is a victory" do 
+			board=["-","-","-","o","o","o","-","-","-"]
+			expect(win.check_win(board)).to eql(true)
+		end
+		it "it returns true if there is a victory" do 
```

Comment:
Leave one empty line around it/specify blocks. This helps to separate the expectations from their conditional logic (contexts for instance).
Checkout this link for more clarification. :point_right: https://github.com/rubocop-hq/rspec-style-guide#empty-lines-around-examples

kindly apply to the rest of the `it` block in your code.

Reply:
perfect